articles/architecture-scenarios/implementation/b2c/b2c-authentication.md

```diff
@@ -35,6 +35,10 @@ useCase:
 
 <%= include('../../_includes/_authentication/_social-authentication.md', { platform: 'b2c' }) %>
 
+## M2M authentication
```

Comment:
Are you suggesting updating B2C Phase 2 planning guidance so that this come before MFA? And if so, would it be as part of Milestone 1 or Milestone 2?

Reply:
I have it in milestone 2, before MFA.  Not sure if you are doing the same or not.